src/components/search-form/search-form.js

```diff
@@ -90,6 +119,74 @@ class SearchForm extends Component {
     return null;
   };
 
+  getSortedDataOptions = () => {
+    const { tagsModel = [] } = this.props;
+    const dataOptions = tagsModel.map(tag => {
+      return {
+        value: tag.label.toLowerCase(),
+        label: tag.label.toLowerCase(),
+      };
+    });
+
+    return sortBy(dataOptions, ['value']);
+  }
+
+  renderTagFilter() {
+    const {
+      tagsModel,
+      searchFilter = {}
+    } = this.props;
+
+    const {
+      tags = ''
+    } = searchFilter;
+
+    const {
+      sections,
+    } = this.state;
+
+    const tagsList = tags ? tags.split(',').map(tag => {
+      return tag.toLowerCase();
+    }) : [];
+
+    tagsList.sort();
+
+    return tagsModel.isLoading
+      ? <Icon icon="spinner-ellipsis" />
+      : (
+        <div data-test-eholdings-tag-filter>
+          <Accordion
+            label={
+              <FormattedMessage id="ui-eholdings.tags" />
```

Comment:
Please, keep it on one line like `label={  }`

Reply:
Thanks Updated!